.tekton/bpfman-agent-pull-request.yaml

```diff
@@ -18,6 +18,10 @@ metadata:
   namespace: ocp-bpfman-tenant
 spec:
   params:
+  - name: build-source-image
+    value: "true"
+  #- name: prefetch-input
```

Comment:
Is there a reason you have these commented out prams here and elsewhere?

Reply:
yes its waiting on konflux fix to address an existing violation